### DISCUSSION TITLE
贊助以後多長時間可以切換到其他翻譯引擎

### DISCUSSION THREAD

**Discussion by ytmeowning** [speech-translator-ext/speech-translator-readme#5] [Q&A] [answered]
我想嘗試除了google以外的翻譯引擎，我贊助了，但是目前來說贊助名單裡看不到我，也沒有在Chrome插件裡找到類似【登錄】的按鈕告訴插件我已經付款了，所以我需要等待多長時間才可以使用付費的功能呢？

**Comment by kappaflow** [comment, marked as answer]
@ytmeowning Thank you for your support! I appreciate it very much. I've already sent the information to you, please check your messages on Boosty.

For subscriptions that require to deploy a personal server (everything above **VIP Supporter**) it may take up to 24 hours to receive personal API key. Shared among subscribers API key for additional translation services gets delivered immediately after subscribing.

Also, please check the **SUBSCRIPTION LEVELS** [here](https://boosty.to/speech-translator-ext) to choose the right support option. In general, additional translation services goes to levels starting from [Advanced Supporter](https://boosty.to/speech-translator-ext/purchase/1967608?ssource=DIRECT&share=subscription_link).
